src/Client.php

```diff
@@ -1,8 +1,8 @@
 <?php
 
-namespace pusher;
+namespace Pusher;
 
-use pusher\Version;
+use Pusher\Version;
```

Comment:
Is the import automatically loaded on use ?

The Version::VERSION is used in the userAgent() function here: https://github.com/pusher/pusher-http-php/blob/new-lib/src/Client.php#L112